src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -7,16 +7,20 @@
 
 namespace Orleans.Serialization
 {
-    internal class OrleansJsonSerializer : IExternalSerializer
+    using Orleans.Providers;
+
+    public class OrleansJsonSerializer : IExternalSerializer
     {
+        public const string UseFullAssemblyNamesProperty = "UseFullAssemblyNames";
+        public const string IndentJsonProperty = "IndentJSON";
         private static JsonSerializerSettings defaultSettings;
         private Logger logger;
-
+        
         /// <summary>
         /// Returns a configured <see cref="JsonSerializerSettings"/> 
         /// </summary>
         /// <returns></returns>
-        internal static JsonSerializerSettings GetDefaultSerializerSettings()
+        public static JsonSerializerSettings GetDefaultSerializerSettings()
```

Comment:
I don't disagree with your reasoning. But doesn't this mean that all storage providers that use `OrleansJsonSerializer` would have to be updated? If so, then I think it is better to make such a change explicit - as a separate PR, and probably targeting 1.4.0, since we are so close to releasing 1.3.0.